### PR TITLE
Finishes reformatting files by formatting all *.java files outside of **/test && **/integrationTest

### PR DESCRIPTION
**Pull request by scrawfor99**
### Description
[Describe what this change achieves]
Follows up #2863

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by reta — APPROVED** [review]

File: src/main/java/org/opensearch/security/auditlog/impl/AuditCategory.java [hunk 1]
```diff
@@ -33,13 +33,8 @@ public enum AuditCategory {
     COMPLIANCE_INTERNAL_CONFIG_WRITE;
 
     public static Set<AuditCategory> parse(final Collection<String> categories) {
-        if (categories.isEmpty())
-            return Collections.emptySet();
+        if (categories.isEmpty()) return Collections.emptySet();
```
Comment (peternied) [inline src/main/java/org/opensearch/security/auditlog/impl/AuditCategory.java:36]: 🫠 I would prefer we always have curly brackets on if statements - can this be tweaked?

File: src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java [hunk 2]
```diff
@@ -400,58 +444,65 @@ public void log(Logger logger) {
...
-                    ", isTransportApiAuditEnabled=" + isTransportApiAuditEnabled +
-                    ", disabledTransportCategories=" + disabledTransportCategories +
-                    ", resolveBulkRequests=" + resolveBulkRequests +
-                    ", logRequestBody=" + logRequestBody +
-                    ", resolveIndices=" + resolveIndices +
-                    ", excludeSensitiveHeaders=" + excludeSensitiveHeaders +
-                    ", ignoredAuditUsers=" + ignoredAuditUsersMatcher +
-                    ", ignoreAuditRequests=" + ignoredAuditRequestsMatcher +
-                    '}';
+            return "Filter{"
```
Comment (peternied) [inline src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java:447]: (Comment on the code) This is a good case for a string builder with format appends for variable pairs. This is terrible performance and heap utilization to allocation each temp substring.

**Review by peternied — APPROVED** [review]
While I've got nit this is great let's merge and we can tackle the small stuff in spectate PRs - great work

File: src/main/java/org/opensearch/security/auditlog/impl/AuditCategory.java (hunk 1 above)
Comment (scrawfor99) [reply src/main/java/org/opensearch/security/auditlog/impl/AuditCategory.java:36]: Yeah, I can look into modifying the formatting file to include that!

Comment (scrawfor99) [reply src/main/java/org/opensearch/security/auditlog/impl/AuditCategory.java:36]: Let me know if you think of any other style guides you would like.

File: src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java (hunk 2 above)
Comment (scrawfor99) [reply src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java:447]: Should we create an issue here?

Comment (peternied) [reply src/main/java/org/opensearch/security/auditlog/config/AuditConfig.java:447]: I don't think so. While there are significant drawbacks to its implementation - it's working code. 

Worth revisiting if they were reports of customer impact

**Pull request merged**